specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2015-11-01-preview/OperationalInsights.json

```diff
@@ -1406,6 +1438,56 @@
       },
       "description": "The list workspaces operation response."
     },
+    "TableProperties": {
+      "properties": {
+        "retentionInDays": {
+          "type": "integer",
+          "format": "int32",
+          "minimum": 30,
+          "maximum": 730,
+          "description": "The data table data retention in days, between 30 and 730. Setting this property to null will default to the workspace retention."
+        }
+      },
+      "description": "Table properties."
+    },
+    "Table": {
```

Comment:
These are common definitions. It's best to use in a shared file or optimally, linked from: https://github.com/Azure/azure-rest-api-specs/blob/fbc094f3097e954f008f8f415a2f95a6dfa3c13b/specification/common-types/resource-management/v1/types.json#L78

Reply:
See my previous request, I will prepare a dedicated PR for this.